utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -650,6 +649,11 @@ function ConvertTo-FormattedJSONParameterObject {
         [string] $BicepParamBlock
     )
 
+    if ([String]::IsNullOrEmpty($BicepParamBlock)) {
```

Comment:
Is this change related to this module testing update?

Reply:
It is. A fix was merged to main and I opened a follow-up PR #2133 to main too - that should resolve this conflict

Reply:
Unfortunately it didn't. Bad boys merge conflicts to be solved.

Reply:
resolved

Reply:
Sorry to insist. I still can't see a reason why an update to the set-modulereadme script should be part of a PR covering the new dependencies approach of a module :/

Reply:
Ah Pardon, I mixed this up with the Pester Test File Update

Reply:
Opened PR #2134 
> Note: You'll likely see this change in other PRs too (until I merge the PR back from main) as this bug effectively blocked the pipelines